src/lib/state.ts

```diff
@@ -637,7 +636,13 @@ export const dumpNode = (node: ZWaveNode, schemaVersion: number): NodeState => {
     return node14;
   }
 
-  return node14 as NodeStateSchema15;
+  if (schemaVersion <= 29) {
+    return node14 as NodeStateSchema15;
+  }
+
+  const node30 = node14 as Partial<NodeStateSchema30>;
+  node30.lastSeen = node.lastSeen;
```

Comment:
Are you using the `as Partial` to stop TS from complaining that `lastSeen` is missing?
I would probably do this instead:
```ts
const node30: NodeStateSchema30 = {
    ...node14,
    lastSeen: node.lastSeen
}
```

Reply:
that's fair, but can we stick with the original pattern for this PR and work on a better one in a separate PR?